guides/batch-compute/01 Hello World.ipynb

```diff
@@ -100,7 +110,11 @@
     "    timeout=600,\n",
     "    retry_count=0,\n",
     ")\n",
-    "async_func.save()"
+    "async_func.save()\n",
+    "\n",
+    "# wait for the function to be ready (will take a few minutes)\n",
+    "print(\"waiting for function to be ready\")\n",
+    "async_func.wait_for_completion()"
    ]
   },
   {
```

Comment:
Sorry for the placement of this comment but I can't put it where I want - above, on line 60, it reads

> Next, we'll create a very basic `hello_world` function that imports a common geospatial Python package and returns a given argument.

Our hello world example does not import anything

Reply:
bah! That had originally been copied from the guide examples (in the docs). I'll fix it.